src/core/i18n/utils.js

```diff
@@ -244,7 +244,15 @@ export function makeI18n(i18nData: I18nConfig, lang: string, _Jed: Jed = Jed) {
   // can type-check all the components that rely on the i18n object.
   // Note: the available locales for tests are controlled in tests/setup.js
   // $FLOW_IGNORE
-  i18n.formatNumber = (number) => new Intl.NumberFormat(lang).format(number);
+  i18n.formatNumber = (number) => {
+    if (Intl && Intl.NumberFormat) {
```

Comment:
Hmm, Flow failed because `Intl` cannot be resolved. I'm not sure how that worked before. It definitely needs a `/* global ... */` definition.